templates/patapsco.html

```diff
@@ -0,0 +1,59 @@
+<p>
+<a href="https://github.com/hltcoe/patapsco">Patapsco</a> is a framework for running cross-language 
+infomration retrieval (CLIR) experiments developed by <a href="https://hltcoe.jhu.edu/">Human Language 
+Technlogy Center of Excellence (HLTCOE) at Johns Hopkins University. 
+</p>
+
+<p>
+To get started with Patapsco, see <a href="https://github.com/hltcoe/patapsco">this guide</a>.
+</p>
+
+<h2 class="underline">Basic Usage</h2>
+
+<p>
+Patapsco specifies the source of the collection via config files or config dictionaries in Python. 
+Please see <a href="https://github.com/hltcoe/patapsco/blob/master/samples/configs/irds_test.yml"> this 
+example config file</a> for reference. 
+</p>
+
+<p>
+For both <kbd>documents</kbd>, <kbd>topics</kbd> and <kbd>scores</kbd> sections, use <kbd>irds</kbd> 
+as the <kbd>format</kbd> in the <kbd>input</kbd> subsection to tell Pataspco to use <kbd>ir_datasets</kbd> 
+and specify the dataset name at <kbd>path</kbd>. The <kbd>lang</kbd> value has to match the language 
```

Comment:
Is there a way to automatically set `lang` for the user from the lang provided by irds? (Or would this be counter to the design of Patapsco?)